events/events.md

```diff
@@ -0,0 +1,32 @@
+## Events Handler
+
+
+EventManager is an interface implemented by the events struct.  Instantiations
+of event manager have 3 methods.  GetEvents returns events found by the manager
+object.  Watch looks for events as they occur and returns them through a channel
+to the caller.  AddEvent is how the caller adds an event to the manager.
+
+
+The events struct holds a slice of event objects in a field called eventlist
```

Comment:
Don't worry about putting implementation details here. As they change we'd need to update here which we will forget to do for sure :) Only documenting the public interface should be fine.